pom.xml

```diff
@@ -61,28 +61,28 @@
         </dependency>
         <dependency>
             <groupId>com.sun.mail</groupId>
-            <artifactId>javax.mail</artifactId>
-            <version>1.6.2</version>
+            <artifactId>jakarta.mail</artifactId>
```

Comment:
did you double check it worked locally? javax to jakarta may break this.

Reply:
I did. It passes `mvn verify`